src/Messenger.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace WMDE\Fundraising\Frontend;
+
+use Swift_Message;
+use Swift_Mime_Message;
+use Swift_Transport;
+
+/**
+ * @licence GNU GPL v2+
+ * @author Kai Nissen < kai.nissen@wikimedia.de >
+ */
+class Messenger {
+
+	private $mailTransport;
+	private $failedRecipients = [];
+	private $operatorAddress;
+
+	public function __construct( Swift_Transport $mailTransport, MailAddress $operatorAddress ) {
```

Comment:
The code in this class is not cohesive. `$operatorAddress` is only used by `constructMessage` and the other two fields are only used by `sendMessage` (and `getFailedRecipients`).

I'm a bit confused by what this ought to be. You can split the two things, and perhaps drop the sendMessage part as its rather simple (even though that means injecting a service from an external library). Alternatively I see how putting them together could make sense if you do something like `sendMessage( MailAddress $sender, MailAddress $receiver, string $subject, string $body )`. When doing that, you decouple the user from Swift stuff altogether.